src/modules/hotkeys/main.ts

```diff
@@ -71,13 +72,35 @@ export const registerHotkeys = async (
     redesignParam?: RedesignParameter,
     LSSM: Vue = window[PREFIX] as Vue
 ) => {
+    const disabledNativeHotkeys: string[] = [];
+    window.pressedKeys = new Proxy(window.pressedKeys ?? {}, {
+        set(obj, key, value) {
+            // we need to get the char from charCode and lowercase because the game uses keyCode instead of key
+            if (
+                typeof key !== 'string' ||
+                !disabledNativeHotkeys.includes(
+                    String.fromCharCode(parseInt(key)).toLowerCase()
+                )
+            )
```

Comment:
This is quite hard to read. I would appreciate some curly brackets and maybe an else statement :)

Reply:
How about that one (although the `else`-keyword is unnecessary):
```ts
if (
    disabledNativeHotkeys.includes(
        String.fromCharCode(parseInt(key.toString())).toLowerCase()
    )
)
    return false;
else return Reflect.set(obj, key, value);
```
![grafik](https://user-images.githubusercontent.com/27238105/204128818-ddcc1cb7-fd04-4b72-b1d2-59f23507e24c.png)

Reply:
Definitely better